libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -527,6 +528,34 @@ private boolean isEvidenceVc(VcStoreItem item) throws CredentialParseException {
         return false;
     }
 
+    private boolean isNonZeroInt(JsonNode node) {
+        return node.isInt() && node.asInt() != 0;
+    }
+
+    private boolean isValidEvidence(VcStoreItem item) {
+        try {
+            JsonNode vcEvidenceNode = getVCClaimNode(item.getCredential(), VC_EVIDENCE);
+            return ((vcEvidenceNode.size() > 0)
+                    && isNonZeroInt(vcEvidenceNode.get(0).path(VC_EVIDENCE_VALIDITY))
```

Comment:
Is this a sufficient check, or should we also check for CIs?

Reply:
I don't know; this wasn't included in the task requirements.